### PR TITLE
DDF-5495 Provide an extension for Table Export behavior

### PR DESCRIPTION
**Pull request by andrewzimmer**
#### What does this PR do?
Forward port of: https://github.com/codice/ddf/pull/5496

#### Who is reviewing it? 
@hayleynorton @zta6 @willwill96 

#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@bdeining
@djblue

#### How should this be tested?
- verify no regression in exporting results from the table view

#### What are the relevant tickets?
Fixes: #5495 

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Review by zta6 — APPROVED** [review]

File: ui-backend/catalog-ui-search-api/src/main/java/org/codice/ddf/catalog/ui/query/utility/CqlQueries.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package org.codice.ddf.catalog.ui.query.utility;
```
Comment (bdeining) [inline ui-backend/catalog-ui-search-api/src/main/java/org/codice/ddf/catalog/ui/query/utility/CqlQueries.java:14]: missing license headers... looks like that needs to be added to this repo

File: ui-backend/catalog-ui-search-api/src/main/java/org/codice/ddf/catalog/ui/query/utility/CqlQueryResponse.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package org.codice.ddf.catalog.ui.query.utility;
```
Comment (bdeining) [inline ui-backend/catalog-ui-search-api/src/main/java/org/codice/ddf/catalog/ui/query/utility/CqlQueryResponse.java:14]: missing license headers... looks like that needs to be added to this repo

**Review by bdeining — APPROVED** [review]

File: ui-backend/catalog-ui-search-api/src/main/java/org/codice/ddf/catalog/ui/query/utility/CqlQueries.java (hunk 1 above)
Comment (bdeining) [reply ui-backend/catalog-ui-search-api/src/main/java/org/codice/ddf/catalog/ui/query/utility/CqlQueries.java:14]: https://github.com/codice/ddf-ui/issues/84#issue-560696518

File: ui-backend/catalog-ui-search-api/src/main/java/org/codice/ddf/catalog/ui/query/utility/CqlQueryResponse.java (hunk 2 above)
Comment (bdeining) [reply ui-backend/catalog-ui-search-api/src/main/java/org/codice/ddf/catalog/ui/query/utility/CqlQueryResponse.java:14]: https://github.com/codice/ddf-ui/issues/84#issue-560696518

**Comment by bdeining** [comment]
hero looks good, no regressions

**Comment by bdeining** [comment]
full approval on https://github.com/codice/ddf/pull/5496

**Pull request merged**